src/govuk/components/character-count/character-count.mjs

```diff
@@ -5,6 +5,42 @@ import '../../vendor/polyfills/Element/prototype/classList.mjs'
 import { closestAttributeValue, extractConfigByNamespace, mergeConfigs, normaliseDataset } from '../../common.mjs'
 import { I18n } from '../../i18n.mjs'
 
+/**
+ * @typedef {object} CharacterCountTranslations
+ * @property {import('../../i18n.mjs').PluralisedTranslations} [charactersUnderLimit]
+ * @property {string} [charactersAtLimit]
+ * @property {import('../../i18n.mjs').PluralisedTranslations} [charactersOverLimit]
+ * @property {import('../../i18n.mjs').PluralisedTranslations} [wordsUnderLimit]
+ * @property {string} [wordsAtLimit]
+ * @property {import('../../i18n.mjs').PluralisedTranslations} [wordsOverLimit]
```

Comment:
We don't define `PluralisedTranslations` in i18n.mjs, only `PluralisedTranslation` (somewhat ironically unpluralised!)

```suggestion
 * @property {import('../../i18n.mjs').PluralisedTranslation} [charactersUnderLimit]
 * @property {string} [charactersAtLimit]
 * @property {import('../../i18n.mjs').PluralisedTranslation} [charactersOverLimit]
 * @property {import('../../i18n.mjs').PluralisedTranslation} [wordsUnderLimit]
 * @property {string} [wordsAtLimit]
 * @property {import('../../i18n.mjs').PluralisedTranslation} [wordsOverLimit]
```